style1.css

```diff
@@ -0,0 +1,97 @@
+@import url('https://fonts.googleapis.com/css2?family=Spinnaker&display=swap');
+*{
+    list-style: none;
```

Comment:
this class will affect all list on-site, and this is not recommended, where the  `list-style: none`  just for one list, not for all list on the site